### PR TITLE
Develop oss connect op output to attr fix

### PR DESCRIPTION
**Pull request by todderasesareddot**
* Added ability to directly connect the output of an operator to an attribute.  Needed to convert from RVType.

* Fixed but where there is an operator output array that is not connected to anything.

* Fixed bug where can't connect to an attribute and also lock that attribute

* Also, turned off verbose output for default value use.  Too noisy.

### CODE REVIEW

File: Python/kraken/core/objects/operators/kl_operator.py [hunk 1]
```diff
@@ -401,7 +401,10 @@ def setRTVal(obj, rtval):
             elif isinstance(obj, Mat44):
                 obj.setFromMat44(rtval)
             elif isinstance(obj, Attribute):
-                obj.setValue(rtval)
+                if ks.isRTVal(rtval):
```
Comment (todderasesareddot) [inline Python/kraken/core/objects/operators/kl_operator.py]: Fixed but where there is an operator output array that is not connected to anything.

File: Python/kraken/core/objects/operators/kl_operator.py [hunk 2]
```diff
@@ -416,10 +419,12 @@ def setRTVal(obj, rtval):
             argConnectionType = arg.connectionType.getSimpleType()
 
             if argConnectionType != 'In':
-                if str(argDataType).endswith('[]'):
-                    for j in xrange(len(argVals[i])):
-                        setRTVal(self.outputs[argName][j], argVals[i][j])
-                elif argName in self.outputs and self.outputs[argName] is not None:
+                if argName in self.outputs and self.outputs[argName] is not None:
```
Comment (todderasesareddot) [inline Python/kraken/core/objects/operators/kl_operator.py]: Added ability to directly connect the output of an operator to an attribute. Needed to convert from RVType.

File: Python/kraken/core/objects/operators/kl_operator.py [hunk 3]
```diff
@@ -401,7 +401,10 @@ def setRTVal(obj, rtval):
             elif isinstance(obj, Mat44):
                 obj.setFromMat44(rtval)
             elif isinstance(obj, Attribute):
-                obj.setValue(rtval)
+                if ks.isRTVal(rtval):
+                    obj.setValue(rtval.getSimpleType())
+                else:
+                    obj.setValue(rtval)
```
Comment (todderasesareddot) [inline Python/kraken/core/objects/operators/kl_operator.py]: Added ability to directly connect the output of an operator to an attribute. Needed to convert from RVType.

**Comment by EricTRocks** [comment]
Looks good.

**Pull request merged**